db/migrate/20160202155708_add_join_token_to_user_groups.rb

```diff
@@ -0,0 +1,5 @@
+class AddJoinTokenToUserGroups < ActiveRecord::Migration
+  def change
+    add_column :user_groups, :join_token, :string
```

Comment:
Should these tokens be unique across the table? For now I figured they don't need to, saves an index and is fine as long as we always check a token against a specific given user_group_id.